BeeKit/Model/BeeminderSpotlightDelegate.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import CoreData
+import CoreSpotlight
+
+class BeeminderSpotlightDelegate: NSCoreDataCoreSpotlightDelegate {
+
+    override func attributeSet(for object: NSManagedObject) -> CSSearchableItemAttributeSet? {
+        if let goal = object as? Goal {
```

Comment:
Prefer swift's guard here?

Reply:
The examples in the docs choose this approach because it scales nicely to multiple different types. It's unclear what other types beeswift might want to index though, so I'm leaning towards "either would be fine"